release-notes/9.0/supported-os.json

```diff
@@ -0,0 +1,277 @@
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "11",
+                        "12"
+                    ]
+                },
+                {
+                    "name": "Fedora",
+                    "lifecycle-policy": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "39",
+                        "40"
+                    ]
+                },
+                {
+                    "name": "openSUSE Leap",
+                    "lifecycle-policy": "https://en.opensuse.org/Lifetime",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "15.5"
+                    ]
+                },
+                {
+                    "name": "Red Hat Enterprise Linux",
+                    "lifecycle-policy": "https://access.redhat.com/support/policy/updates/errata/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "8",
+                        "9"
+                    ]
+                },
+                {
+                    "name": "SUSE Enterprise Linux",
+                    "lifecycle-policy": "https://www.suse.com/lifecycle/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "39",
+                        "40"
+                    ]
+                },
+                {
+                    "name": "Ubuntu",
+                    "lifecycle-policy": "https://wiki.ubuntu.com/Releases",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "20.04",
+                        "22.04",
+                        "23.10",
+                        "24.04"
+                    ]
+                }
+            ]
+        },
+        {
+            "family": "Windows",
+            "distributions": [
+                {
+                    "name": "Nano Server",
+                    "lifecycle-policy": "https://learn.microsoft.com/windows-server/get-started/windows-server-release-info",
+                    "architectures": [
+                        "x64"
+                    ],
+                    "supported-cycles": [
+                        "Version 1809+"
```

Comment:
I'm not a too big fan of using wildcards but would prefer more explicit listing like in the other cases (i.e. Ubuntu). This would allow people to easier see when one is dropped.

Reply:
I agree. I got lazy at the end. I'll try to expand these.